package/base-files/files/etc/profile

```diff
@@ -41,3 +41,12 @@ in order to prevent unauthorized SSH logins.
 --------------------------------------------------
 EOF
 fi
+
+service() {
+if [ -f "/etc/init.d/$1" ]; then
+        "/etc/init.d/$@"
+else
+        ls "/etc/init.d"
+fi
```

Comment:
right now the list is implicit and wont even cause a bad error code. please drop the implicit list and return an error instead